src/components/Aardbei/Aardbei.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import './Aardbei.css';
+
+
+function Aardbei({buttonType,buttonDisabled,children,clickHandler}) {
```

Comment:
Je hebt voor elke fruitsoort een apart button component gemaakt. Echter ziet dit eruit als een doorgeefluik. Er wordt hiermee geen code/logica gereduceerd. 

Zoals ik in mijn eerdere comment al zei, zou je dit component kunnen uitbreiden dat de logica voor het verlagen/ophogen van de counter en de bijbehorende buttons. Op deze manier kun je code hergebruiken voor elke fruitsoort. Dit zorgt er dan op zichzelf wél voor dat er code gereduceerd is